### ISSUE TITLE
Usage

### ISSUE THREAD

**Issue by v1rtl** [codejamninja/git-sync-all#2]
There is nothing said in README on how to use it. Would be nice to see some examples

**Comment by clayrisser** [comment]
Yeah sorry, it's not finished yet. I just started the project 12 hours ago.

**Comment by lgg** [comment]
@codejamninja please, can you add basic examples and documentation? It's hard to read source code for contributing to docs. :pray:

**Comment by clayrisser** [comment]
Sure, I'll try and get some basic docs up. The project is still in the early stages, but some basic things are working.
